libs/vkd3d/device.c

```diff
@@ -8230,6 +8231,25 @@ static void d3d12_device_caps_init_shader_model(struct d3d12_device *device)
             INFO("Enabling support for SM 6.7.\n");
             device->d3d12_caps.max_shader_model = D3D_SHADER_MODEL_6_7;
         }
+
+        /* SM6.8 adds:
+         * - SV_StartInstanceLocation / SV_StartVertexLocation (required)
+         *   - Trivially maps to BaseVertex and BaseInstance, respectively
+         * - WaveSizeRange (required)
+         *   - Completely replaces WaveSize at the DXIL level
+         *   - Specifies a range of subgroup sizes that the shader can operate
+         *     with, as well as a preferred subgroup size.
+         * - Expanded Comparison Sampling
+         *   - Basically Grad/Bias operands for Dref instructions
+         * - Work graphs
+         *   - uhh...
+         */
+        if (device->d3d12_caps.max_shader_model == D3D_SHADER_MODEL_6_7 &&
+                (vkd3d_config_flags & VKD3D_CONFIG_FLAG_ENABLE_EXPERIMENTAL_FEATURES))
```

Comment:
Is there a reason to put this behind experimental flag? We can enable SM 6.7 unconditionally now. We just lack AdvancedTexturing ops. This commit should probably be moved last so it cannot break bisects.